scripts/components/material/guild.js

```diff
@@ -44,7 +44,7 @@ function EmojiRow(props) {
   return (
     props.emoji.map((emoji) => (
       <MaterialEmoji baseSize={32} key={emoji.id} {...emoji}
-                     boxClassName={clsx(classes.cell)} />
+        boxClassName={clsx(classes.cell)} />
```

Comment:
If you want to fix this, move baseSize's line down to its own one, then this can be inline with it